package.json

```diff
@@ -10,17 +10,14 @@
     "install-if-no-packages": "node bin/install-if-no-packages.js",
     "clean": "rimraf ./dist",
     "prebuild": "npm run -s install-if-deps-outdated",
-    "build:core": "npm run -s i18n && cross-env NODE_ENV=production webpack",
-    "postbuild": "npm run -s i18n:php && npm run -s i18n:pot",
+    "build": "npm run clean && npm run -s i18n && cross-env NODE_ENV=production webpack",
     "start": "webpack --watch",
     "i18n:js": "cross-env NODE_ENV=production babel js -o /dev/null",
-    "i18n:php": "pot-to-php ./languages/action-scheduler-admin.pot ./languages/action-scheduler-admin.php action-scheduler-admin",
-    "i18n:pot": "grunt makepot",
+    "i18n:pot": "wp --allow-root i18n make-pot . languages/action-scheduler-admin.pot --include=\"js,includes\" --exclude=\"dist,node_modules\" --slug=action-scheduler-admin",
```

Comment:
The inclusion of `wp` means the execution needs to be in a WordPress environment. My VVV environment lets me SSH into the container to execute the `build` command, but it does add an extra step. This would affect an eventual CI or automated build step. Is there another way?

Reply:
> The inclusion of wp means the execution needs to be in a WordPress environment. My VVV environment lets me SSH into the container to execute the build command, but it does add an extra step. This would affect an eventual CI or automated build step.

How do you run `npm run makepot` in the WP core repo locally? The [`build:core` script](https://github.com/woocommerce/woocommerce/blob/trunk/package.json#L21) uses it and it does not require that the repo be in the `plugins` folder in a WP install. It's been a few years since I used VVV, but IIRC, you should be able to run this without SSHing into the container.